test/vars/UtilSpec.groovy

```diff
@@ -435,4 +435,62 @@ class UtilSpec extends JenkinsPipelineSpecification {
         then:
         result == "/workspacefolder/kiegroup_projectA"
     }
+
+    def "[util.groovy] getNextSnapshotMicro"() {
```

Comment:
small thing but can you rename the different tests `getNextVersion[...]` instead of `getNextSnapshot[...]` ?
as the tested method is `getNextVersion`